openapi.go

```diff
@@ -275,9 +319,13 @@ func (route *Route[ResponseBody, RequestBody, Params]) RegisterParams() error {
 		for i := range typeOfParams.NumField() {
 			field := typeOfParams.Field(i)
 			var params []ParamOption
-			example, _ := field.Tag.Lookup("description")
+			example, _ := field.Tag.Lookup("example")
 			if example != "" {
-				params = append(params, ParamExample("example", example))
+				parsedValue, err := parseExampleFromTag(example, field.Type)
+				if err != nil {
+					return fmt.Errorf("parsing example tag: %s", err)
+				}
+				params = append(params, ParamExample("example", parsedValue))
```

Comment:
I'm not sure I follow your issue you exactly. Could you possibly send a snipped of code that was failing?

Reply:
Oh I see. Examples is a map. So for this: 

```golang
option.Query("name", "Filter by name", param.Example("cat name", "felix"), param.Nullable(), param.Example("cat name", "bob")),
```

you end up with:

```json
"examples": {
        "cat name": {
	      "value": "felix"
	},
}
```

where you want the ability to do:

```json
"examples": {
"my example": {
	"value": [
		1,
		2,
		3
	]
}
}
```

Reply:
Okay yeah now I see. Sorry it's been a awhile since I've been here.